apps/examples/tickless_test/tickless_test_main.c

```diff
@@ -0,0 +1,108 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define NTHREADS	5
+#define LOOP		3
+
+#define MSEC_PER_SEC	1000 /* 1s = 1000ms */
+
+#define MSEC_PER_NSEC	(1 / 1000000) /* 1ns = (1/1000000)ms */
+
+#define USEC_PER_MSEC	1000 /* 1ms = 1000us */
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <time.h>
+
+/****************************************************************************
+ * Private Data
+ ****************************************************************************/
+
+static unsigned int thread;
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+static void *thread_func(void *arg)
+{
+	unsigned int duration = (unsigned int)arg;
+	unsigned int cur_thread = thread;
```

Comment:
i think it's better to use getpid() ? instead of believing the global variable "thread"